biomejs/biome
# feat: add more ignored files

## Summary
fix #869
Let me know if I need to add/remove (more) file(s)




## Test Plan
- Check if newly added files are applicable to ignore formatting
- Check if all ignored files are listed in `/guides/how-biome-works/#known-files` (`pnpm start` in `website` dir)
